src/Core/Content/Product/SalesChannel/Sorting/ProductSortingEntity.php

```diff
@@ -52,6 +52,7 @@ public function createDalSorting(): array
         $sorting = [];
 
         $fields = $this->fields;
+        array_push($fields, $this->getFallbackSortingField());
```

Comment:
can you add a small unit test for this?

Reply:
Can you move the fallback sorting into the ProductListingFeaturesSubscriber so it's more obvious that we add a sorting :)

Inside the Struct, it feels like really hidden magic happening :)

https://github.com/shopware/platform/blob/d1ada242838f32dc5f9910aff7a63cf5b58053b2/src/Core/Content/Product/SalesChannel/Listing/ProductListingFeaturesSubscriber.php#L267

Reply:
I understand. I moved it there, hopefully that works for you :)